### ISSUE TITLE
Fix "no visible bindings" messages

### ISSUE THREAD

**Issue by rplzzz** [JGCRI/gcamdata#368]
The package checker produces an enormous number of messages that look like this:
```
find_chunks: no visible binding for global variable ‘name’
find_chunks: no visible binding for global variable ‘disabled’
find_chunks: no visible binding for global variable ‘x’
...
```
There are so many of these that when something actually fails, it's not so easy to find it in the output.  These messages are resulting from the nonstandard evaluation in dplyr, and the straightforward, if a little inelegant, fix is to do some dummy assignments at the beginning of the function that uses them.  Should we just bite the bullet and do this so that our package checks are easier to make sense of?

**Comment by bpbond** [comment]
Sigh. Yeah, probably, agreed. The one thing I'd say is let's prefix the definition block with a constant marker so it's easy to take them out/change automated-ly in the future, if we want. E.g.
```R
# Define pipeline NSE variables so don't get package build messages
name <- disabled <- x <- NULL
```

**Comment by rplzzz** [comment]
Sounds reasonable.  Also, butting it right at the beginning of each function will help.

I'll do some work on this in the background.  Also, we can start looking for it in the new modules as people submit them.

**Comment by bpbond** [comment]
Fixed in #413
